pkg/virt-controller/services/template.go

```diff
@@ -379,7 +381,9 @@ func getMemoryOverhead(domain v1.DomainSpec) *resource.Quantity {
 	overhead.Add(resource.MustParse("8Mi"))
 
 	// Add video RAM overhead
-	overhead.Add(resource.MustParse("16Mi"))
+	if domain.Devices.AutoattachGraphicsDevice == nil || *domain.Devices.AutoattachGraphicsDevice == true {
```

Comment:
just a thought. it might be nice to have a convenience function in the api for this... something like. vmi.HasGraphics()